### PR TITLE
Update documentation how to build nitro locally, add instructions for MacOS

### PR DESCRIPTION
**Pull request by anodar**
- updated go version to 1.20
- updated rust version to 1.72.1

### CODE REVIEW

File: arbitrum-docs/node-running/how-tos/build-nitro-locally.md [hunk 1]
```diff
@@ -10,16 +10,38 @@ import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-pa
 
 <PublicPreviewBannerPartial />
 
-This how-to is based on [Debian 11.7 (arm64)](https://cdimage.debian.org/cdimage/archive/11.7.0/arm64/iso-cd/debian-11.7.0-arm64-netinst.iso) and [Ubuntu 22.04 (amd64)](https://releases.ubuntu.com/22.04.2/ubuntu-22.04.2-desktop-amd64.iso).
+This how-to is based on [Debian 11.7 (arm64)](https://cdimage.debian.org/cdimage/archive/11.7.0/arm64/iso-cd/debian-11.7.0-arm64-netinst.iso) and [Ubuntu 22.04 (amd64)](https://releases.ubuntu.com/22.04.2/ubuntu-22.04.2-desktop-amd64.iso), and [MacOS Ventura 13.4](https://developer.apple.com/documentation/macos-release-notes/macos-13_4-release-notes).
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/how-tos/build-nitro-locally.md:13]:
```suggestion
This how-to is based on [Debian 11.7 (arm64)](https://cdimage.debian.org/cdimage/archive/11.7.0/arm64/iso-cd/debian-11.7.0-arm64-netinst.iso), [Ubuntu 22.04 (amd64)](https://releases.ubuntu.com/22.04.2/ubuntu-22.04.2-desktop-amd64.iso), and [MacOS Ventura 13.4](https://developer.apple.com/documentation/macos-release-notes/macos-13_4-release-notes).
```

File: arbitrum-docs/node-running/how-tos/build-nitro-locally.md [hunk 2]
```diff
@@ -30,54 +52,93 @@ git submodule update --init --recursive --force
...
+
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+nvm install 16.19
+nvm use 16.19
+```
+
 ### 4. Configure Rust [1.66.1](https://www.rust-lang.org/tools/install)
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/how-tos/build-nitro-locally.md:74]:
```suggestion
### 4. Configure Rust [1.72.1](https://www.rust-lang.org/tools/install)
```

**Comment by TucksonDev** [comment]
Thanks for this @anodar 🙏

File: arbitrum-docs/node-running/how-tos/build-nitro-locally.md [hunk 3]
```diff
@@ -10,16 +10,38 @@ import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-pa
 
 <PublicPreviewBannerPartial />
 
-This how-to is based on [Debian 11.7 (arm64)](https://cdimage.debian.org/cdimage/archive/11.7.0/arm64/iso-cd/debian-11.7.0-arm64-netinst.iso) and [Ubuntu 22.04 (amd64)](https://releases.ubuntu.com/22.04.2/ubuntu-22.04.2-desktop-amd64.iso).
+This how-to is based on [Debian 11.7 (arm64)](https://cdimage.debian.org/cdimage/archive/11.7.0/arm64/iso-cd/debian-11.7.0-arm64-netinst.iso) / [Ubuntu 22.04 (amd64)](https://releases.ubuntu.com/22.04.2/ubuntu-22.04.2-desktop-amd64.iso), and [MacOS Ventura 13.4](https://developer.apple.com/documentation/macos-release-notes/macos-13_4-release-notes).
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/how-tos/build-nitro-locally.md:13]:
```suggestion
This how-to assumes that you're running one of the following operating systems:
 - [Debian 11.7 (arm64)](https://cdimage.debian.org/cdimage/archive/11.7.0/arm64/iso-cd/debian-11.7.0-arm64-netinst.iso)
 - [Ubuntu 22.04 (amd64)](https://releases.ubuntu.com/22.04.2/ubuntu-22.04.2-desktop-amd64.iso)
 - [MacOS Ventura 13.4](https://developer.apple.com/documentation/macos-release-notes/macos-13_4-release-notes).
```

File: arbitrum-docs/node-running/how-tos/build-nitro-locally.md [hunk 4]
```diff
@@ -30,54 +52,93 @@ git submodule update --init --recursive --force
...
+  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+sudo service docker start
 ```
 
-### 6. Configure Go [1.19.5](https://github.com/moovweb/gvm)
+#### For [MacOS](https://docs.docker.com/desktop/install/mac-install/)
+
+Depending on whether your Mac has Intel Chip or Apple silicon, download corresponding
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/how-tos/build-nitro-locally.md:111]:
```suggestion
Depending on whether your Mac has an Intel processor or Apple silicon, download the corresponding disk image from [Docker](https://docs.docker.com/desktop/install/mac-install/), and move it into your Applications folder.
```

File: arbitrum-docs/node-running/how-tos/build-nitro-locally.md [hunk 5]
```diff
@@ -30,54 +52,93 @@ git submodule update --init --recursive --force
...
+disk image from [docker](https://docs.docker.com/desktop/install/mac-install/), and move it to applications folder.
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/how-tos/build-nitro-locally.md:112]:
```suggestion
```

**Comment by symbolpunk** [comment]
Thank you @anodar and @TucksonDev !

**Pull request merged**